automatic/videoder/legal/VERIFICATION.txt

```diff
@@ -0,0 +1,20 @@
+VERIFICATION
+
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+Package can be verified like this:
+
+1. Go to
+
+   x32: https://www.videoder.com/download/videoder-for-windows?arch=32
+   x64: https://www.videoder.com/download/videoder-for-windows?arch=64
+
+   to download the installer.
+
+2. You can use one of the following methods to obtain the SHA256 checksum:
+   - Use powershell function 'Get-FileHash'
+   - Use Chocolatey utility 'checksum.exe'
+
+   checksum32: C4653C5562E13E202C7F6CECA8F5F87BB984C8A758A02A5A0CCD62047792EDCE
```

Comment:
Is it OK to have such hash hard-coded for the current version?

Reply:
Yeah, its the purpose of the file to have it hardcoded there. AU updates it each time.